coral-schema/src/main/java/com/linkedin/coral/schema/avro/SchemaUtilities.java

```diff
@@ -618,6 +632,10 @@ static Schema makeNonNullable(Schema schema) {
   }
 
   static Schema makeNullable(Schema schema) {
```

Comment:
why not remove this method and change all the calling references to `makeNullable(schema, false)` ? I feel that makes the code easier to follow.